src/new-block/render.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
This file shows a lot of linting issues.

![image](https://github.com/user-attachments/assets/a20cbe19-079c-4284-a901-8bfa9a4abb35)

The Gutenberg docs show using `wp_kses_data` on the attributes: https://github.com/WordPress/gutenberg-examples/blob/trunk/blocks-jsx/10-dynamic-block/template.php#L13